examples/DynamicCantileverCase/dynamic_cantilever_helper.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+
+class DynamicCantileverVibration:
+    def __init__(
+        self,
+        base_length,
+        base_area,
+        moment_of_inertia,
+        youngs_modulus,
+        density,
+        mode,
+        end_velocity_z=0.0,
+    ):
+        self.base_length = base_length
+        self.end_velocity_z = end_velocity_z
+
+        assert isinstance(mode, int)
+        assert mode >= 0
+        assert mode < 5
+        self.mode = mode
+
+        betas = np.array([0.596864, 1.49418, 2.50025, 3.49999]) * np.pi / base_length
+        self.beta = betas[mode]
+        self.omega = (self.beta ** 2) * np.sqrt(
+            youngs_modulus
+            * moment_of_inertia
+            / (density * base_area * base_length ** 4)
+        )
+
+        np_mode_l = self._compute_nonparametrized_mode(
+            base_length, base_length, self.beta
+        )
+        self.param = end_velocity_z / (-1j * self.omega * np_mode_l)
+
+        # if mode > 1:
+        #     step = 1.0 / (mode - 1)
+        #     test_x = np.arange(0.0, 1.0 + step, step=step)[1:] * base_length
+        #     param_mat = np.zeros([mode, mode], dtype="complex128")
+        #     target_mat = np.zeros([mode, 1], dtype="complex128")
+        #
+        #     for i, x in np.ndenumerate(test_x):
+        #         param_mat[i, :] = self._compute_nonparametrized_mode(x, base_length, self.betas)
+        #         target_mat[i] = 0.0
+        #
+        #     param_mat[-1, :] = -1j * self.omegas * np_modes_l
+        #     target_mat[-1] = end_velocity_z
+        #
+        #     soln_mat = la.solve(param_mat, target_mat)
+        #     self.params = soln_mat.ravel()
+
+    def get_initial_position_profile(self, x_coords):
```

Comment:
`x_coords` is `position`? Same here please more specific. Also is it depend on the x position?

Reply:
[3e1ca62](https://github.com/GazzolaLab/PyElastica/pull/173/commits/3e1ca62f44482de6dc719256fd47975ea3aa5037)

Reply:
[58ea0df](https://github.com/GazzolaLab/PyElastica/pull/173/commits/58ea0df586e4e373d5ec560fa7d3a9abeee8668a)